specification.md

```diff
@@ -353,7 +353,8 @@ Represents data of a round held at the competition.
 | Attribute | Type | Description |
 | --- | --- | --- |
 | `id` | `String` | The round identifier of the form `{eventId}-r{roundNumber}`. *Note: this is a valid [`ActivityCode`](#activitycode).* |
-| `format` | `"1"\|"2"\|"3"\|"a"\|"m"` | The round format. Look [here](https://github.com/thewca/worldcubeassociation.org/blob/master/WcaOnRails/db/seeds/formats.seeds.rb) for the list of all the WCA formats. |
+| `format` | `"1"\|"2"\|"3"\|"a"\|"m"` | The round format. Look [here](https://github.com/thewca/worldcubeassociation.org/blob/master/db/seeds/formats.seeds.rb) for the list of all the WCA formats. |
+| `roundType` | `"0"\|"1"\|"2"\|"3"\|"b"\|"c"\|"d"\|"e"\|"f"\|"g"\|"h"` | The round type. Look [here](https://github.com/thewca/worldcubeassociation.org/blob/master/db/seeds/round_types.seeds.rb) for the list of all the WCA round types. (In essence, a Round Type describes this round in relation to other rounds of this competition.) |
```

Comment:
It seems a little redundant to expose the "combined" roundTypes (c,d,e,g,h), since they're rendered the same as f,1,2,3, and q. Maybe we should not include these in the API?

(Personally I'd love to get rid of those roundTypes entirely, but that's a larger discussion)

Reply:
I'd guess the reason the round type hasn't been included so far is that it can be determined based on round number and whether it has a cutoff, so this information is somewhat redundant. I don't have strong opinion though, if there's a good use case for including the summarised field, that's ok.

Reply:
It's because of this PR https://github.com/thewca/worldcubeassociation.org/pull/9262. We use wcif to render the events tab and translating the activity code to a i18n string is really annoying and would involve a lot of `if roundNumber == x && maxRounds == y return z ` with this you can just do `i18n.t(rounds.${round.roundType}.cellName)`

Reply:
You could have a single function that takes the relevant info and generates the round type id, to encapsulate all the logic in a single place, then invoke it in the interpolation.

Reply:
sure, but why go through the trouble of converting to wcif and then "unconverting" the wcif back to the round type by duplicating code from the ruby side into js if you can include that data in the output (either through the wcif or as additional data to the react component if this is a controversial change)

Reply:
It's mostly about avoiding duplication, but again, I don't have strong opinion :)